superset/assets/utils/common.js

```diff
@@ -39,8 +39,19 @@ export function getParamFromQuery(query, param) {
   return null;
 }
 
-export function getLink(baseUrl, params) {
-  return baseUrl + '?' + params.join('&');
+export function getQueryLink(query, callback) {
```

Comment:
I can see your point :) both options are fine, let's treat this comment as resolved.
To explain my opinion:
```
f1() {
  q = {
    a:a,
    b:b,
    c:c,
    d:d,
  }
  do_smth(query)
}

f2() {
  q = {
    a:a,
    b:b,
    c:c,
    d:d,
  }
  do_smth(query)
}

do_smth() {
  send_request(query)
}

=================  vs =========================

f1() {
  do_smth(a, b, c, d)
}

f2() {
  do_smth(a, b, c, d)
}

do_smth() {
  send_request({
    a:a, 
    b:b,
    c:c,
    d:d,
  })
}
```